app/auto_users.go

```diff
@@ -48,6 +48,11 @@ func (a *App) CreateBasicUser(client *model.Client4) *model.AppError {
 		if resp.Error != nil {
 			return resp.Error
 		}
+		store.Must(a.Srv.Store.User().VerifyEmail(ruser.Id, ruser.Email))
```

Comment:
Why are this changes added here?

Reply:
There was a merge conflict so these new changes were merged.

Reply:
I'm fixing this